.gitignore

```diff
@@ -23,4 +23,6 @@ build/
 dist/
 
 # IDE Config Directories
-.idea/
\ No newline at end of file
+.idea/
+
+terraform_example/*
```

Comment:
can be `terraform_example/` if you want to exclude the directory too, this just makes sure it's always empty. But if that's intended leave it as is :)